Packs/FeedFeedly/pack_metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Feedly",
+    "description": "Import Articles from Feedly with enriched IOCs",
+    "support": "partner",
+    "currentVersion": "1.0.0",
+    "author": "Feedly",
+    "url": "https://feedly.com/i/landing/threatIntelligence",
+    "email": "support@feedly.com",
+    "categories": [
+        "Data Enrichment & Threat Intelligence"
+    ],
+    "tags": [
+        "Threat Intelligence Management"
```

Comment:
@Mathieu4141 if this feature is available to non paying customers please add "Free feed" as a tag and add the pack in the "FreeFeed" Pack metadata as well (https://github.com/demisto/content/blob/master/Packs/FreeFeeds/pack_metadata.json)

If not, ignore this message.